README.md

```diff
@@ -2,41 +2,43 @@
 
 UPSTAGE is a **U**niversal **P**latform for **S**imulating
 **T**asks and **A**ctors with **G**raphs and **E**vents built atop
-[*SimPy 4*][simpy].
+[__`SimPy 4`__][simpy-repo].
 
 ## What is UPSTAGE for?
 
-The Universal Platform for Simulating Tasks and Actors with Graphs and Events (**UPSTAGE**) library is a Python framework for creating robust, behavior-driven Discrete Event Simulations (DES). The primary goal of UPSTAGE is to enable the quick creation of simulations at any desired level of abstraction with built-in data recording, simulation integrity and runtime checks, and assistance for the usual pitfalls in custom discrete-event simulation: interrupts and cancellations. It is designed is to simplify the development process for simulation models of *complex systems of systems*.
+The _Universal Platform for Simulating Tasks and Actors with Graphs and Events_ (__UPSTAGE__) is a Python framework for creating robust, behavior-driven Discrete Event Simulations (DES). The primary goal of UPSTAGE is to enable the quick creation of simulations at any desired level of abstraction with built-in data recording, simulation integrity and runtime checks, and assistance for the usual pitfalls in custom discrete-event simulation: interrupts and cancellations. It is designed is to simplify the development process for simulation models of *complex systems of systems*.
```

Comment:
I like library in there because it sounds weird if you read it as "The UPSTAGE is". 

Maybe just "UPSTAGE is a Python.." since the acronym is defined in the header.